torch/utils/_config_module.py

```diff
@@ -493,12 +493,61 @@ def codegen_config(self) -> str:
         """Convert config to Python statements that replicate current config.
         This does NOT include config settings that are at default values.
         """
+
+        # additional imports required
+        imports = set()
+
+        def get_module_name(func, add_dot):
+            module_name = func.__module__
+            if module_name == "builtins":
+                module_name = ""
+            if func.__module__ == "_warnings":
+                module_name = "warnings"
+            if add_dot and module_name != "":
```

Comment:
How about this one?  Can we turn relative import into absolute import?

Reply:
> How about this one? Can we turn relative import into absolute import?

@desertfire  The "dot" here is the dot between "loggings.warn". It's not for the absolute/relative import. I have this flag here because if the module is builtin, then we should just have "print", but not ".print".

Reply:
I see. Then I think spelling out the full `builtins.print` makes more sense here. As a debugging tool, it's fine that the generated file is a little verbose.

Reply:
> I see. Then I think spelling out the full `builtins.print` makes more sense here. As a debugging tool, it's fine that the generated file is a little verbose.

@desertfire We cannot spell out the full `builtins.print`, because `builtins.print` is not runnable. You'll get `NameError: name 'builtins' is not defined` error. 

btw I made some minor changes to the PR to check `isinstance(v, __module__)`. If `v` is a partial function, for example, `functools.partial(<function collect_fwd_graph_outputs at 0x7fbfe640e050>, fwd_outputs=set())`, it won't have a module. 

In this case, we'll just output the string above. It's not runnable, I can fix that later in a followup PR.